augur/utils.py

```diff
@@ -571,11 +582,13 @@ def run_shell_command(cmd, raise_errors = False, extra_env = None):
     except FileNotFoundError as error:
         print_error(
             """
+            {out}
             Unable to run shell commands using {shell}!
 
             Augur requires {shell} to be installed.  Please open an issue on GitHub
             <https://github.com/nextstrain/augur/issues/new> if you need assistance.
             """,
+            out   = out,
```

Comment:
Again, I don't think `out` will be defined if an exception was raised here?

Reply:
```suggestion
```